src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -540,6 +540,15 @@ public function thisProductHasAnImageWithACode(ProductInterface $product, $image
     }
 
     /**
+     * @Given /^(this product)'s price is "([^"]+)"$/
+     */
+    public function thisProductPriceIs(ProductInterface $product, $price)
+    {
+        $product->getFirstVariant()->setPrice($this->getPriceFromString($price));
```

Comment:
`getFirstVariant()` should not be used anymore, we should always use `defaultVariantResolver` as we do e.g. [here](https://github.com/tuka217/Sylius/blob/fdef0bf31a94eb6b3b34c4394101870fe72af188/src/Sylius/Behat/Context/Setup/ProductContext.php#L497).

PS. We should remove `getFirstVariant` method from `Product` in a separate PR.